CHANGES.rst

```diff
@@ -1,6 +1,18 @@
 Release notes
 =============
 
+1.18.0 (2023-03-03)
+-------------------
+
+- feature: Add return path option for amazon ses emails (`PR#381 <https://github.com/scrapinghub/spidermon/pull/381>`_)
+- bug: Fix `BaseStatMonitor` failing in Scrapy Cloud when settings were provided as strings (`PR#378 <https://github.com/scrapinghub/spidermon/pull/378>_`)
+- feature: Add setting `SPIDERMON_FIELD_COVERAGE_SKIP_IF_NO_ITEM` to allow skipping `FieldCoverageMonitor` if not items were scraped (`PR#372 <https://github.com/scrapinghub/spidermon/pull/372>_`)
+- feature: Add `Fallback Actions`. `Action` now allows to define `fallback` field that may contain an additional `Action` to be executed if an exception is raised during the main action (`PR#365 <https://github.com/scrapinghub/spidermon/pull/365>_`)
```

Comment:
It would be great if we could link “Fallback Actions” and “Actions” here to the corresponding documentation sections.

Also, mind that for monospace font in reStructuredText you need 2 `: ``` ``like this`` ```.